### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by shlomoh19** [aravindvnair99/AravindVNair99#1] [Q&A] [answered]
Hi, I have an issue
At the order export level CVC, transfers in and out must be typed as such - not as "Subscriptions" and "Redemptions". so How to include these new types to this export?

3. These new type of orders should be included in the APIs. How?

4. These new type of orders should be reflected in the different alerts sent by mail. How?

I used Angular, node mysql

**Comment by aravindvnair99** [comment, marked as answer]
Hey @shlomoh19!

Thanks for reaching out! I, unfortunately, didn't understand the context of your questions. I didn't understand what you meant by `order export level CVC`, `"Subscriptions" and "Redemptions"`, etc. Could you please provide more details so that I can assist you further?
